prettyprinter-ansi-terminal/src/Prettyprinter/Render/Terminal/Internal.hs

```diff
@@ -242,6 +270,18 @@ panicStyleStackNotFullyConsumed len
            "end of rendering (there should be only 1). Please report" ++
            " this as a bug.")
 
+-- | Various kinds of colors that can be used in a terminal
+data AnsiColor
+  -- | A color from the standard palette of 16 colors (8 colors by 2 color intensities). Many terminals allow the palette colors to be customised
```

Comment:
Does the `-- |` before the `=` render correctly? I always avoided this style for that reason, but maybe it changed?